type/__apt_source/files/source.list.template

```diff
@@ -1,15 +1,9 @@
-#!/bin/sh
+#!/bin/sh -e
 set -u
```

Comment:
move `-u` to shebang too?

Reply:
Well, I tried to keep the changes to a minimum, to be honest. The only reason that I even touched the shebang line is, because I sort of assumed the policy for `-e` is something that we've kept from `cdist`.

I personally feel `-u` is a bit of an overkill, but - again - tried to keep the changes to a minimum.

Similarly, I don't expect anyone running this internal template as a standalone script - so I wouldn't worry about that.

However at the end of the day this will be a core type, so maybe erring on the side of caution isn't such a bad idea. So, I'd say let's move both to the shebang line then; any objections?